content/forgero-compat/src/main/resources/data/forgero/packs/betterend/material/umbralith.json

```diff
@@ -0,0 +1,101 @@
+{
+  "type": "STONE",
+  "name": "umbralith",
+  "parent": "stone_base",
+  "container": {
+    "type": "HOST",
+    "id": "betterend:umbralith"
+  },
+  "properties": {
+    "grouped_attributes": [
+      {
+        "context": "COMPOSITE",
+        "order": "BASE",
+        "operation": "ADDITION",
+        "attributes": [
+          {
+            "id": "stone-mining-level-composite",
+            "type": "MINING_LEVEL",
+            "value": 2
+          },
+          {
+            "id": "stone-durability-composite",
+            "type": "DURABILITY",
+            "value": 190
+          },
+          {
+            "id": "stone-rarity-composite",
+            "type": "RARITY",
+            "value": 45
+          },
+          {
+            "id": "stone-weight-composite",
+            "type": "WEIGHT",
+            "value": 40
+          }
+        ]
+      },
+      {
+        "context": "UPGRADE",
+        "attributes": [
+          {
+            "id": "stone-rarity-upgrade",
+            "type": "RARITY",
+            "order": "BASE",
+            "operation": "ADDITION",
+            "category": "ALL",
+            "value": 22
+          },
+          {
+            "id": "stone-weight-upgrade",
+            "type": "WEIGHT",
+            "order": "BASE",
+            "operation": "ADDITION",
+            "category": "ALL",
+            "value": 20
+          },
+          {
+            "id": "stone-attack_damage-offensive-upgrade",
+            "type": "ATTACK_DAMAGE",
+            "operation": "ADDITION",
+            "category": "OFFENSIVE",
+            "value": 2
+          },
+          {
+            "id": "stone-mining_speed-offensive-upgrade",
+            "type": "MINING_SPEED",
+            "operation": "ADDITION",
+            "category": "OFFENSIVE",
+            "value": 1.75
+          },
+          {
+            "id": "stone-durability-offensive-upgrade",
+            "type": "DURABILITY",
+            "operation": "ADDITION",
+            "category": "OFFENSIVE",
+            "value": 95
+          },
+          {
+            "id": "stone-durability-utility-upgrade",
```

Comment:
Stones don't have any utility slots that they can be used in. As highlited by this PR: https://github.com/SigmundGranaas/forgero/pull/961